internal/cli/events/list.go

```diff
@@ -79,6 +79,7 @@ func (opts *ListOpts) newEventListOptions() *atlas.EventListOptions {
 // mongocli atlas event(s) list [--projectId projectId] [--orgId orgId] [--page N] [--limit N] [--minDate minDate] [--maxDate maxDate]
 func ListBuilder() *cobra.Command {
 	opts := &ListOpts{}
+	opts.Template = listTemplate
```

Comment:
[q} Can these be written in the same way as https://github.com/mongodb/mongocli/pull/379/files#diff-7dfd7a5005571cf7b5e5368ae7a939bcR62, to make it more consistent?

Reply:
We don't have a call to our own `PreRunE` func, so it's a bit ugly since that function is thought to be called with that
as in, to use it here it would be something like `opts.InitOutput(cmd.OutOrStdout(), listTemplate)(),`

I'm thinking we need to make `PreRunE` and pass the validations functions instead of relaying on it doing the project id validation but that will be another day

For this le me know if switching to  `_ = opts.InitOutput(cmd.OutOrStdout(), listTemplate)()` makes sense (also ignore error as that never return an error, is just there to fulfil the signature of `PreRunE`

Reply:
Ah got it, I think that last suggestion makes sense, imo (but I wouldn't block on it if you want to change the structure soon)